### DISCUSSION TITLE
Mandatory taxes after partial refund?

### DISCUSSION THREAD

**Discussion by p49gm** [pretix/pretix#2450] [Ideas] [answered]
The possibility of voluntary partial refunds is a great addition to pretix and helps organizers a lot in the pandemic, thanks! I think it could be even more helpful.

Currently, for the remaining amount a new invoice _including_ taxes (VAT) is generated. To my knowledge the inclusion of VAT can not be altered in pretix. 

I don't think that VAT is mandatory in all cases, i.e. according to German tax laws after the cancellation of an event the attendee is not getting anything in return for his money, so the "donated" amount should not be subject to taxes. That's even true for involuntary cancellation fees (German link: https://www.pkf-fasselt.de/news/nachrichten-blog/artikel/umsatzsteuerliche-behandlung-von-stornogebuehren-678.html).

Perhaps this is a feature to be considered.

Best regards
Andreas

**Comment by raphaelm** [comment, marked as answer]
Cancellation fees currently use the tax rate configured for payment fees at settings > fees. Yes, this is wrong and confusing and we plan a change, but for now, that's how you can fix it :)

**Reply by p49gm** [reply]
Oh, thank you very much, you saved my day! :)
